Favor remote execution over katello agent (#444)
theforeman/foreman-documentation: guides/doc-Content_Management_Guide/topics/QuickStart.adoc

Uploading Enabled Repositories Report
----
. Check the `/etc/yum.repos.d/redhat.conf` and ensure that the appropriate repositories have been enabled.

==== Install a package

Packages can now be installed while logged in to the Content Host

----
 # yum install screen
----

To manage packages through the {ProjectServer}, configure `Remote Execution`.
For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs] in the  _Managing Hosts_ guide.

> Once configured, use this procedure to install a package from the {ProjectServer} web UI.

Reviewer: ```suggestion
After you have configured remote execution, use this procedure to install a package from the {ProjectServer} web UI.
```
Once to mean when confuses our translations allegedly :)

.Procedure
. In the {Project} web UI, navigate to *Hosts* > *Content Hosts*.

> . Click the name of the host which was registered above.

Reviewer: ```suggestion
. Click the name of the registered host.
```

. Click the *Packages* tab.
. From the dropdown, click *Actions*.

> . Select *Package Install* from the dropdown.

Reviewer: ```suggestion
. Select *Package Install* from the list.
```

> . Type *screen* into the text field.

Reviewer: ```suggestion
. Type `screen` into the text field.
```

> . From the dropdown next to the *Perform* button, select `via remote execution`.

Reviewer: ```suggestion
. From the dropdown next to the *Perform* button, select `via remote execution`.
```
Could you share a screenshot so I know the right UI label element to reference according to the IBM Style Guide :)

> A `Job Invocation` will be created to perform the package installation and report the status back to the web UI.

Reviewer: ```suggestion

```
Generally, here the advice is to tell them how they can find a positive status on the job.
